src/objects/logo-group/logo-group.scss

```diff
@@ -0,0 +1,15 @@
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/ms';
+
+.o-logo-group {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: (-1 * ms.step(3));
```

Comment:
This addresses my earlier feedback, but I notice now that this component needs a lot of padding to avoid horizontal scrollbars. Maybe it makes sense to add the padding back to the parent element (instead of requiring utilities), we should just make sure to include adequate comments explaining why... for example, "we add generous padding to balance out the whitespace differentiating brands" or something to that effect.

Reply:
I removed the negative margin, curious what you think of it now?

Reply:
@dromo77 Did you push that change? I still see the negative margin.

Reply:
No 😆  should be there now.